src/test/test_ep_cred_client.c

```diff
@@ -140,16 +152,42 @@ test_run()
 		assert(rc == 0);
 
 		input = crt_req_get(rpc);
-		if (i == 0)
-			input->pi_delay = 1;
-		else
+
+		if (i == 0) {
```

Comment:
to take it to the extreme, you can delay (test.tg_credits - 1) rpcs, that way you only make room for one rpc from the wait queue.

Reply:
yeah i thought about that, but then it would slow down test significantly